openedx/core/djangoapps/api_admin/api/v1/views.py

```diff
@@ -0,0 +1,52 @@
+"""
+API Views.
+"""
+
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+
+from edx_rest_framework_extensions.authentication import JwtAuthentication
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.exceptions import NotFound, PermissionDenied
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.generics import ListAPIView
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from openedx.core.djangoapps.api_admin.api.v1 import serializers as api_access_serializers
+from openedx.core.djangoapps.api_admin.models import ApiAccessRequest
+
+
+class ApiAccessRequestView(ListAPIView):
+    """
+    List Api Access Requests.
```

Comment:
Can we make it more descriptive? 
e.g. just example request, Query Parameter, return structure.